File: sdkwork-backend-sdk-go/types/chat_options.go
package types


type ChatOptions struct {
	Model string `json:"model"`
	Temperature float64 `json:"temperature"`
	TopP float64 `json:"topP"`
	TopK int `json:"topK"`
	MaxTokens int `json:"maxTokens"`
	StopSequences []string `json:"stopSequences"`
	FrequencyPenalty float64 `json:"frequencyPenalty"`
	PresencePenalty float64 `json:"presencePenalty"`
}

File: sdkwork-backend-sdk-csharp/Models/ChatOptions.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class ChatOptions
    {
        public string? Model { get; set; }
        public double? Temperature { get; set; }
        public double? TopP { get; set; }
        public int? TopK { get; set; }
        public int? MaxTokens { get; set; }
        public List<string>? StopSequences { get; set; }
        public double? FrequencyPenalty { get; set; }
        public double? PresencePenalty { get; set; }
    }
}
